examples/console/package.json

```diff
@@ -8,7 +8,7 @@
   "dependencies": {
     "@jupyterlab/codemirror": "^0.5.0",
     "@jupyterlab/console": "^0.5.1",
-    "@jupyterlab/default-theme": "^0.5.1",
+    "@jupyterlab/theming": "^0.5.1",
```

Comment:
Opps missed that this is an example, ignore that comment. Examples definitely need to include the themeing.